Create ui-c8y-1019-7-3-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md (#1768)
SoftwareAG/c8y-docs: content/change-logs/ui-c8y-1019-7-3-unable-to-switch-tabs-when-dashboard-has-two-widgets-with-dashboard.md

---
date: ""

> title: Fix inability to switch tabs with multiple widgets using dashboard date context

Reviewer: ```suggestion
title: Switching tabs with multiple widgets using the dashboard date context works smoothly
```

> title: Unable to switch tabs when dashboard has two widgets with dashboard date context (#5682) [GRAFT][release/cd] (#5707)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fix Issue Switching Tabs with Dashboard Date Context Widgets
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58128
version: 1019.7.3
---

> Unable to switch tabs when dashboard has two widgets with dashboard date context (#5682) [GRAFT][release/cd] (#5707)

Author: The Claude AI suggested to use the following body:
```suggestion
Fixed an issue where users were unable to switch between dashboard tabs when the dashboard contained two widgets using dashboard date contexts. Users can now smoothly transition between tabs regardless of widgets using dashboard date contexts.
```
